indico/web/forms/fields.py

```diff
@@ -830,6 +830,8 @@ def __init__(self, *args, **kwargs):
         self.protected_object = kwargs.pop('protected_object')(kwargs['_form'])
         get_acl_message_url = kwargs.pop('acl_message_url', None)
         self.acl_message_url = get_acl_message_url(kwargs['_form']) if get_acl_message_url else None
+        self.iacl_count = len([x for x in self.protected_object.protection_parent.get_access_list(skip_managers=True)
+                               if x not in [entry.principal for entry in self.protected_object.acl_entries]])
```

Comment:
The second one shouldn't have the `[]` around it. That way you just iterate over the items and count the number of elements yielded by the generator expression. The first one creates a list just to get its length and then throws it away again